account_invoice_inter_company/models/account_invoice.py

```diff
@@ -286,6 +288,10 @@ def action_cancel(self):
                 for inter_invoice in self.sudo().search(
                         [('auto_invoice_id', '=', invoice.id)]):
                     inter_invoice.signal_workflow('invoice_cancel')
+                    inter_invoice.write({
+                        'supplier_invoice_number': '',
```

Comment:
s/'supplier_invoice_number': '',/'supplier_invoice_number': False,

Reply:
@bealdav ,
It's done.